tests/unit/core/reducers/test_abuse.js

```diff
@@ -0,0 +1,164 @@
+import abuseReducer, {
+  SEND_ADDON_ABUSE_REPORT,
+  initialState,
+  loadAddonAbuseReport,
+  sendAddonAbuseReport,
+} from 'core/reducers/abuse';
+import { dispatchClientMetadata, fakeAddon } from 'tests/unit/amo/helpers';
+import { createFakeAddonAbuseReport } from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('initializes properly', () => {
+      const state = abuseReducer(initialState, { type: 'UNRELATED_ACTION' });
+      expect(state).toEqual({ bySlug: {} });
```

Comment:
I think it would be better to make this assertion:

```js
expect(state).toEqual(initialState);
```
Otherwise you'd have to update the test every time it changes which doesn't seem necessary.